sp800-63-3/sec3_definitions.md

```diff
@@ -104,7 +104,7 @@ A process that allows for the conveyance of identity and authentication informat
 A set of attributes that uniquely describe a person within a given context.
 
 #### Identity Assurance Level (IAL)
-A metric describing degree of confidence that the applicant’s claimed identity is their real identity.
+An ordinal that conveys the degree of confidence that the applicant’s claimed identity is their real identity.
```

Comment:
It's not a metric, true, but not an ordinal either: we don't say IAL 2nd. This will do for now, but let's think of a better word. Something like 'category'?